src/android/app/src/main/res/values/strings.xml

```diff
@@ -481,13 +484,19 @@
     <string name="memory_petabyte">PB</string>
     <string name="memory_exabyte">EB</string>
 
-    <!-- Theme Modes -->
+    <!-- System theme -->
     <string name="change_theme_mode">Change Theme Mode</string>
     <string name="theme_mode_follow_system">Follow System</string>
     <string name="theme_mode_light">Light</string>
     <string name="theme_mode_dark">Dark</string>
+
+    <!-- Material You theme -->
     <string name="material_you">Material You</string>
-    <string name="material_you_description">Use system colors across the app</string>
+    <string name="material_you_description">Use system colors across the app (Will override (static) theme color option)</string>
+
+    <!-- Static theme color -->
+    <string name="static_theme_color">Change Theme Color</string>
+    <string name="static_theme_color_description">Change the main theme color of the app without the use of Material You (Requires Material You to be disabled)</string>
```

Comment:
```suggestion
    <string name="static_theme_color_description">Change the color theme of the app's menus</string>
```
Mentioning the Material You thing three times is redundant.

Reply:
It was meant so users dont get confused about the setup

I guess i can have used different strings, but since the setup page is getting removed, I'll just revert